### DISCUSSION TITLE
Tutorials for NLP Contest

### DISCUSSION THREAD

**Discussion by ZoranPandovski** [mindsdb/mindsdb#4214] [Announcements]
:loudspeaker:  We are launching the new contest of Tutorials for NLP! :bear: 

Natural Language Processing inside databases opens vast possibilities for developers, and we need your help to democratize this!
If you have a brilliant idea of how to use this in real life - > create a tutorial and get fabulous prizes :trophy:  such as:

- [JBL Wireless Earbuds](https://www.amazon.com/JBL-225TWS-Wireless-Earbud-Headphones/dp/B08BR45QPR/ref=sr_1_1) and [MindsDB t-shirt](https://uploads-ssl.webflow.com/62a8755be8bcc86e6307def8/6329bc04a5b921b6c9c3ca8b_rewards-1.png) for every author of validated tutorial!
- [3-in-1 Wireless Charging Stand](https://www.amazon.com/mophie-Universal-Wireless-Charging-AirPods/dp/B08NTVQ8HZ/ref=sr_1_1_sspa) if your tutorial reaches 1k minutes read/watched!
- Get coverage for T&E to San Francisco to attend the MindsDB AI conference and have dinner with MindsDB founders!
- Remote Job Opportunities :handshake: for the best authors - part-time or full-time!

Please [check the contest page](https://mindsdb.com/in-database-nlp-tutorials-contest?utm_medium=community&utm_source=slack&utm_campaign=22q4-huggingface-preview) :bookmark: for more details!

**Comment by C0staTin** [comment]
We are happy to share the results summary pf the NLP Tutorials competition:

MindsDB hosted an NLP competition from 12th December 2022 till 29th January 2023 for the Hugging Face Integration with MindsDB, giving contributors the opportunity to contribute written or video tutorials about unique use cases for using HuggingFace with MindsDB.

We would like to thank those who participated in the HuggingFace Integration with MindsDB NLP competition, your submissions were amazing! Congratulations to Balaji Seetharaman for winning the prize for the highest ‘number of minutes watched/read time’ for an article! 

We received 1 Youtube tutorial and 8 written tutorials:

- [Balaji Seetharaman](https://github.com/bala-ceg) - [Generate Auto Emoji Reactions to Elon Musk’s Tweets Using MindsDB](https://techwithbala.medium.com/282b6def732f) & [Unlock Crypto Market Buying Signals using MindsDB](https://www.linkedin.com/posts/techwithbala_mindsdb-ai-nlp-activity-7025055455538245632-lW0S?utm_source=share&utm_medium=member_desktop)
- [Prathik Shetty](https://github.com/prathikshetty2002) - [Hate speech detection using Mindsdb & Hugging face](https://medium.com/@prathikkshetty15/b322a5b356a3)
- [Munyoudoum Pav](https://github.com/munyoudoum) - [Detect languages using Mindsdb in-database NLP](https://www.youtube.com/watch?v=4cdSvpB-5Go)
- [Parthiv Makwana](https://github.com/parthiv11) - [Sentiment analysis using YouTube comments](https://medium.com/@parthivmakwana11/e89dcdfd13b0)
- [Sriniketh J](https://github.com/srini047) - [Product Reviews Sentiment Analysis](https://medium.com/@srini047/dd4f742a1475)
- [Zubeen](https://github.com/syedzubeen) - [Summarizing Text with In-Database NLP using MindsDB](https://medium.com/@zubeen/90f2da5f2b00)
- [Kamil Tyborowski](https://github.com/ktyborowski) - [How to predict purchase intent using SQL](https://dev.to/ktyborowski/how-to-predict-purchase-intent-using-sql-2p9c)
- [Ephraim](https://github.com/EphraimX) - [NLP: Real and Fake News Detection Using MindsDB and MongoDB](https://dev.to/ephraimx/nlp-real-and-fake-news-detection-using-mindsdb-and-mongodb-50gi)
